test/dotnet-build.Tests/BuildOutputTests.cs

```diff
@@ -176,6 +176,25 @@ public void SettingVersionSuffixFlag_ShouldStampAssemblyInfoInOutputAssembly()
             informationalVersion.Should().BeEquivalentTo("1.0.0-85");
         }
 
+        [Fact]
+        public void BuildGlobbingMakesAllRunnable()
+        {
+            var testInstance = TestAssetsManager.CreateTestInstance("AppWithAppDependency")
+                                                .WithLockFiles();
+
+            var cmd = new BuildCommand(string.Format("*{0}project.json", Path.DirectorySeparatorChar), skipLoadProject: true)
```

Comment:
This can all be one fluent expression:

```C#
new BuildCommand(string.Format("*{0}project.json", Path.DirectorySeparatorChar), skipLoadProject: true)
  .WithWorkingDirectory(testInstance.TestRoot)
  .Execute()
  .Should()
  .Pass();
```

Note you don't need to `ExecuteWithCapturedOutput` if you aren't going to do anything with the captured output.